lib/streamlit/elements/media.py

```diff
@@ -247,10 +258,12 @@ def video(
 
             * ``None`` (default): No subtitles.
 
-            * A string, bytes, or Path: File path to a subtitle file in ``.vtt`` or ``.srt`` formats, or
-              the raw content of subtitles conforming to these formats.
-              If providing raw content, the string must adhere to the WebVTT or SRT
-              format specifications.
+            * A string, bytes, or Path: File path to a subtitle file in
```

Comment:
Haha. Ironically, I was actually choosing to unify around `-` instead of `*` since I had determined it was more common in the code base; I just didn't go so far as to update all existing ones. Although, when I looked again, they are pretty much evenly tied--about 200 instances of each. 😅